packages/client/test/util/wasmCrypto.spec.ts

```diff
@@ -18,9 +17,11 @@ import {
   waitReady,
   sha256 as wasmSha256,
 } from '@polkadot/wasm-crypto'
+import { secp256k1 } from 'ethereum-cryptography/secp256k1'
```

Comment:
Not totally a blocker, but it's a bit semi-beautiful that we have a mixture of these `.js` and non-`.js` imports for the `ethereum-cryptography` imports.